### PR TITLE
Select factors to determine how polygons are covered

### PR DESCRIPTION
**Pull request by mikefab**
screen shot 2018-03-27 at 10 48 36 am
screen shot 2018-03-27 at 10 48 42 am

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -81,37 +86,53 @@ class App extends Component {
...
-    let layerName = e.target.getAttribute('name')
-    let currentStatus = this.state.map.getLayoutProperty(layerName, 'visibility')
-    this.state.map.setLayoutProperty(layerName, 'visibility', nextState[currentStatus])
+    let layerName = e.target.getAttribute('nombre')
 
-    if (nextState[currentStatus] === 'none') {
-      e.target.classList.remove('active')
+    let currentStatus = this.state.map.getLayer(layerName)
+    let currentOpacityValue = this.state.map.getPaintProperty('regions', 'fill-opacity')
+    console.log(e.target.checked, '!!!!')
```
Comment (rdebeasi) [inline src/App.js]: Don't forget to remove the console log! :smiley:

**Comment by rdebeasi** [comment]
Looks good so far!

I noticed that `check-panel` and `helpers` aren't checked in. Would you mind adding these?

It also might be worth seeing if we can get rid of the "trash" variable and implement the checkbox UI shown in the wireframes. I'd be happy to pair with you on this if that would be helpful.

**Comment by mikefab** [comment]
They're checked in. Thank you for reminding me.

**Comment by mikefab** [comment]
Corrected HDI 
screen shot 2018-03-27 at 8 07 36 pm

**Comment by mikefab** [comment]
"It also might be worth seeing if we can get rid of the "trash" variable and implement the checkbox UI shown in the wireframes. I'd be happy to pair with you on this if that would be helpful."

That would be great. Thank you.

**Comment by rdebeasi** [comment]
Closing this PR because this code is now included in #73.

**Pull request closed without merging**